src/LaborMarket/LaborMarket.sol

```diff
@@ -340,7 +282,7 @@ contract LaborMarket is
         );
 
         require(
-            reviewSignals[submissionId][msg.sender],
+            reviewSignals[requestId][msg.sender],
```

Comment:
It will need to be updated to check against an existing balance of signaling once the future PR supports volume-based signaling.